lib/clients/aws-client/iam-client.js

```diff
@@ -77,6 +77,7 @@ module.exports = class IAMClient extends AbstractAwsClient {
     _generateIAMRoleName(skillName) {
```

Comment:
Also no need to add the ternary check because if it's shorter then 39, original string will be returned.

Reply:
>Could you add what does this 39 mean to the comment for this function? Like what's the maximum length for a role name, and how it comes to 39 maximum for the skill name?
>
>Please refer to https://github.com/alexa/ask-cli/blob/develop/lib/clients/aws-client/s3-client.js#L263-L266 as a similar description.

Sure, will add a comment.

>Also no need to add the ternary check because if it's shorter then 39, original string will be returned.

I believe we need it because on line 81 we are inserting the truncatedSkillName and only the validSkillName if its less than <=39 characters, so it still has to be assigned to the truncatedSkillName variable.

Reply:
>Also no need to add the ternary check because if it's shorter then 39, original string will be returned.

Removed the ternary operator.